common/app/templates/inlineJS/blocking/applyRenderConditions.scala.js

```diff
@@ -64,6 +64,47 @@
         }).join(' ');
     }
 
+    /*
+        This is a shortened version of shouldSeeReaderRevenue() from
+        user-features.js. Since we are blocking rendering at this time we
+        can't/ don't want to online all required JS from this module.
+    */
+    function hideReaderRevenue() {
+        var cookies = document.cookie.split('; ');
+        var contribution = {
+            cookie: 'gu.contributions.contrib-timestamp',
+            fullfilled: false,
+            check: function(value) {
+                var now = new Date().getTime();
+                var lastContribution = new Date(value).getTime();
+                var diffDays = Math.ceil((now - lastContribution) / (1000 * 3600 * 24));
+
+                return diffDays <= 180;
+            },
+        };
+        var member = {
+            cookie: 'gu_paying_member',
+            fullfilled: false,
+            check: function(value) {
+                return value === 'true';
+            },
+        };
+
+        for(var i = 0; i < cookies.length; ++i) {
```

Comment:
it looks to be faster to use a regex on the cookie, based on https://jsperf.com/get-cookie-value-regex-vs-array-functions. we also check the cookie at the bottom of the page too. maybe we should store the value for that 2nd read?